docs/sphinx/developers/examples/TiledReaderWriter.java

```diff
@@ -184,17 +177,18 @@ private void cleanup() {
    * $ java TiledReaderWriter input-file.oib output-file.ome.tiff 256 256
    * @throws IOException
    * @throws FormatException
+   * @throws ServiceException 
+   * @throws DependencyException 
    */
-  public static void main(String[] args) throws FormatException, IOException {
+  public static void main(String[] args) throws FormatException, IOException, DependencyException, ServiceException {
     int tileSizeX = Integer.parseInt(args[2]);
     int tileSizeY = Integer.parseInt(args[3]);
     TiledReaderWriter tiledReadWriter = new TiledReaderWriter(args[0], args[1], tileSizeX, tileSizeY);
     // initialize the files
     boolean initializationSuccess = tiledReadWriter.initialize();
```

Comment:
As reported by Travis, the method is now `void` so the `boolean initializationSuccess =` should be removed.